rpcs3/Emu/Memory/vm.cpp

```diff
@@ -783,6 +783,7 @@ namespace vm
 			{
 				std::make_shared<block_t>(0x00010000, 0x1FFF0000), // main
 				std::make_shared<block_t>(0x20000000, 0x10000000), // user
+				std::make_shared<block_t>(0x40000000, 0x30000000), // rsx contexts
```

Comment:
I put this down to 0x10000000 because it seemed to force sonic unleashed to be unable to vm_map with it taking so much memory, may have to be looked into what actually is used by ps3 then